rma/i18n/es.po

```diff
@@ -10,43 +10,52 @@ msgstr ""
 "PO-Revision-Date: 2022-01-17 12:39+0000\n"
 "Last-Translator: xavigutipujol <jgutierrez@pyming.com>\n"
 "Language-Team: \n"
-"Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+<<<<<<< HEAD
```

Comment:
These conflicts must be solved

Reply:
This one escaped from my local pre-commit run... Fixed